arxiv/base/config.py

```diff
@@ -1,46 +1,51 @@
 """Flask configuration."""
 
+from typing import Optional
 import os
+from urllib.parse import urlparse
 
 SERVER_NAME = None
 
-EXTERNAL_URLS = [
-    ("twitter", os.environ.get("ARXIV_TWITTER_URL",
-                               "https://twitter.com/arxiv")),
-    ("blog", os.environ.get("ARXIV_BLOG_URL",
-                            "https://blogs.cornell.edu/arxiv/")),
-    ("wiki", os.environ.get("ARXIV_WIKI_URL",
-                            "https://confluence.cornell.edu/display/arxivpub/"
-                            "arXiv+Public+Wiki")),
-    ("accessibility", os.environ.get("ARXIV_ACCESSIBILITY_URL",
-                                     "mailto:web-accessibility@cornell.edu")),
-    ("library", os.environ.get("ARXIV_LIBRARY_URL",
-                               "https://library.cornell.edu")),
-    ("acknowledgment", os.environ.get(
-        "ARXIV_ACKNOWLEDGEMENT_URL",
-        "https://confluence.cornell.edu/x/ALlRF"
-    )),
-]
-"""External URLs, configurable via environment variables."""
+A11Y_URL = os.environ.get("ARXIV_ACCESSIBILITY_URL",
+                          "mailto:web-accessibility@cornell.edu")
 
-ARXIV_URLS = [
-    ("help", "/help"),
-    ("contact", "/help/contact"),
-    ("search_box", "/search"),
-    ("search_advanced", "/search/advanced"),
-    ("account", "/user"),
-    ("login", "/user/login"),
-    ("logout", "/user/logout"),
-    ("home", "/"),
-    ("pdf", "/pdf/<arxiv:paper_id>"),
+URLS = [
+    ("help", "/help", "arxiv.org"),
+    ("contact", "/help/contact", "arxiv.org"),
+    ("search_box", "/search", "arxiv.org"),
+    ("search_advanced", "/search/advanced", "arxiv.org"),
+    ("account", "/user", "arxiv.org"),
+    ("login", "/user/login", "arxiv.org"),
+    ("logout", "/user/logout", "arxiv.org"),
+    ("home", "/", "arxiv.org"),
+    ("pdf", "/pdf/<arxiv:paper_id>", "arxiv.org"),
+    ("twitter", "/arxiv", "twitter.com"),
+    ("blog", "/arxiv", "blogs.cornell.edu"),
+    ("wiki", "/display/arxivpub/arXiv+Public+Wiki", "confluence.cornell.edu"),
+    ("library", "/", "library.cornell.edu"),
+    ("acknowledgment", "/x/ALlRF", "confluence.cornell.edu")
```

Comment:
Since this hasn't been merged yet, can you add the following URLs?

@blueprint.route("help_identifier", "/help/arxiv_identifier", "arxiv.org")
@blueprint.route("help_mathjax", "/help/mathjax/", "arxiv.org")
@blueprint.route("help_trackback", "/help/trackback/","arxiv.org")
@blueprint.route("help_social_bookmarking", "/help/social_bookmarking","arxiv.org")

# Not sure how to do static
@blueprint.route("arxiv_css", "/css/arXiv.css?v=20170424","static.arxiv.org")

# Not sure how to do the favicon
@blueprint.route("favicon", /favicon.ico, "arxiv.org")

# deprecated? 
@blueprint.route("", "/find","arxiv.org")

# anti-robot honnypot 
@blueprint.route("ignore_me", "/IgnoreMe","arxiv.org")